packages/wpcom-checkout/src/field.tsx

```diff
@@ -102,25 +107,15 @@ const Label = styled.label< { disabled?: boolean } >`
 const Input = styled.input< {
 	isError?: boolean;
 	icon?: ReactNode;
+	country?: string;
```

Comment:
What is `country` used here for? I don't see it used anywhere below...

Reply:
Great catch, this was from the original fix where Input styles were conditionally applied based on whether the country was set etc. Removed.